### ISSUE TITLE
toc-location not working in HTML?

### ISSUE THREAD

**Issue by maxdrohde** [quarto-dev/quarto-cli#471]
When I set `toc-location: left`, the table of contents still shows up on the right.

See the code and output below:

## Minimal Example

```
---
title: "Test"
toc: true
format:
  html:
    toc-location: left
---

# Section 1

## Section 2
```

## Output
<img width="1017" alt="Screen Shot 2022-03-27 at 13 56 25" src="https://user-images.githubusercontent.com/25734232/160296382-900482d7-f847-4002-8ae8-f958155d7160.png">

## System details

Quarto: Version: 0.9.128
MacOS: 12.3
Rendered in RStudio: 2022.02.0 Build 443

**Comment by dragonstyle** [comment]
The way we have `roc-location` implemented right now only functions inside of a website (not in standalone HTML documents). I will take a look at fixing that this morning!

**Comment by dragonstyle** [comment]
I've taken a closer look at this- it is a bit more involved than I anticipated as we don't currently reserve room in our grid system for the left based TOC in the case of standalone document. In the case of websites, we are able to utilize the fact that sidebar navigation reserves space on the left, making it a bit easier since that space is already allocated. In the case of a standalone document, we'll need to allocate the additional space and then be aware of other page layout elements (e.g. margin content) that is using the right side margin- this is all basically taken care of in the case of websites.

I won't be able to get to this today, but will keep it open for me to take a look at when I revisit the grid system and page layout.

**Comment by dragonstyle** [comment]
Ok, I've add support for `roc-location: left` in standalone html documents. It should simply work as expected (and in fact we now default the TOC location to the left in order to make the experience of using margin elements nicer). Give it a try in the latest build and lmk how things work for you! (should be available in v0.9 build 215).

**Comment by maxdrohde** [comment]
Yes it works for me -- thanks for implementing this! I agree, default on the left makes more sense with margin elements.
